README.rst

```diff
@@ -1,29 +1,71 @@
 Gammapy
 =======
 
-A Python Package for Gamma-ray Astronomy
+A Python Package for Gamma-ray Astronomy.
+
+Gammapy is an open-source Python package for gamma-ray astronomy built on Numpy, Scipy and Astropy. It is used as core
+library for the Science Analysis tools of the Cherenkov Telescope Array (CTA), recommended by the H.E.S.S.
+collaboration to be used for Science publications, and is already widely used in the analysis of existing gamma-ray
+instruments, such as MAGIC, VERITAS and HAWC.
 
 * Webpage: https://gammapy.org
 * Documentation: https://docs.gammapy.org/
 * Code: https://github.com/gammapy/gammapy
-* License: BSD-3
+* .. image:: https://anaconda.org/conda-forge/gammapy/badges/license.svg
+    :target: https://github.com/gammapy/gammapy/blob/master/LICENSE.rst
+    :alt: Licence
+
 
 .. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
     :target: http://www.astropy.org/
 
-.. image:: http://mybinder.org/badge.svg
-    :target: https://mybinder.org/v2/gh/gammapy/gammapy-webpage/master?urlpath=lab/tree/overview.ipynb
+.. image:: https://anaconda.org/conda-forge/gammapy/badges/installer/conda.svg
+
+.. image:: https://anaconda.org/conda-forge/gammapy/badges/downloads.svg
+
+..
+    ToDo: Add Binder in the release process
+    .. image:: http://mybinder.org/badge.svg
+        :target: https://mybinder.org/v2/gh/gammapy/gammapy-webpage/master?urlpath=lab/tree/overview.ipynb
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4701488.svg?style=flat
+    :target: https://doi.org/10.5281/zenodo.4701488
+
+.. image:: https://archive.softwareheritage.org/badge/swh:1:dir:02e8a702ef6c9558a9f96d99bf6b9f41b5edcd34/
+    :target: https://archive.softwareheritage.org/swh:1:dir:02e8a702ef6c9558a9f96d99bf6b9f41b5edcd34;origin=https://github.com/gammapy/gammapy;visit=swh:1:snp:11294b68ac9dcb4aeefaa784e7fb7afba7d61402;anchor=swh:1:rev:89f67e3014c3673704ea6eeb8ad3ad0b844a3426
+
+Contributing Code, Documentation, or Feedback
++++++++++++++++++++++++++++++++++++++++++++++
+The Gammapy Project is made both by and for its users, so we welcome and encourage contributions of many kinds. Our
+goal is to keep this a positive, inclusive, successful, and growing community by abiding with the
+`Gammapy Community Code of Conduct <https://gammapy.org/CoC.html>`_.
+
+The Gammapy project uses a mechanism known as a `Developer Certificate of Origin (DCO) <https://github.com/apps/dco>`_
+to manage the legal aspect of contributions. The DCO is a legally binding statement that asserts that you are the
+creator of your contribution, and that you wish to allow Gammapy to use your work to cite you as contributor.
+Acknowledgement of this permission is described in the `Developer guide` of the official documentation.
```

Comment:
Replace by a real link to the dev docs: https://docs.gammapy.org/dev/development/intro.html

Reply:
done